app/models/performance_statistics.rb

```diff
@@ -80,15 +87,40 @@ def candidate_status_counts
       .to_a
   end
 
+  def candidate_status_total_counts
+    candidate_status_counts.group_by { |row| row['status'] }.map do |status, rows|
+      { 'status' => status, 'count' => rows.map { |r| r['count'] }.sum }
+    end
+  end
+
   def total_submitted_count
     total_candidate_count(except: %i[never_signed_in unsubmitted_not_started_form unsubmitted_in_progress])
   end
 
+  def apply_again_submitted_count
+    total_candidate_count(except: %i[never_signed_in unsubmitted_not_started_form unsubmitted_in_progress], phase: :apply_2)
+  end
+
   def ended_without_success_count
-    total_candidate_count(only: %i[ended_without_success])
+    total_candidate_count(only: %i[rejected_by_default ended_without_success])
   end
 
   def accepted_offer_count
     total_candidate_count(only: %i[pending_conditions recruited offer_deferred])
   end
+
+  def apply_again_accepted_offer_count
+    total_candidate_count(only: %i[pending_conditions recruited offer_deferred], phase: :apply_2)
+  end
+
+  def rejected_by_default_count
+    total_candidate_count(only: %i[rejected_by_default])
+  end
+
+  def percentage_of_providers_onboarded
+    @percentage_of_providers_onboarded ||= begin
+      counts = Provider.group(:sync_courses).count
```

Comment:
Not sure if this is the correct definition of onboarded. I would consider it to be the "providers with open courses" stat from here: https://www.apply-for-teacher-training.service.gov.uk/support/performance/course-stats. Maybe we don't need it here?